Tools/Scripts/filter-build-webkit

```diff
@@ -43,6 +43,7 @@ use File::Basename;
 use FindBin;
 use lib $FindBin::Bin;
 use Getopt::Long;
+use Term::ANSIColor qw(color);
```

Comment:
Is this Perl module installed by default on macOS?

if not, it needs to loaded differently in case it is missing.

Reply:
It's the same module as other scripts load.